### PR TITLE
Polish Help Text

### PR DESCRIPTION
**Pull request by tajmone**
Fix typos and polish English of the text shown via `--help`.

### CODE REVIEW

**Comment by wshito** [comment]
@tajmone Thank you for your contribution!  I will review it when I have time.  I am quite busy with my work at the moment so please give me some time.

**Comment by tajmone** [comment]
Don't worry, it's nothing urgent, just aesthetic improvements.

File: src/CommandOptions.mjs [hunk 1]
```diff
@@ -21,61 +21,61 @@ export const makeConfig = (argv) => {
...
-  it with the '-o' option.
+  Splits a single html document generated by Asciidoctor into
+  multiple chunked html files.
+  The default output directory is 'html_chunks'.  You can override
+  it via the '-o' option.
 
-  The default splits are made by preamble, parts, and chapters.
-  You can specify the extraction level with the '--depth' option.
+  The default splitting criteria is by preamble, parts, and chapters.
+  You can specify the extraction level via the '--depth' option.
```
Comment (wshito) [inline src/CommandOptions.mjs]: I prefer the original here.

File: src/CommandOptions.mjs [hunk 2]
```diff
@@ -21,61 +21,61 @@ export const makeConfig = (argv) => {
...
-  so 3:2 means chapter 3 is extracted down to 2 levels (i.e.
-  section level). You can use a hyphen to specify the range
-  of chapters to set as chapFrom-chapTo:level, so 1-3:5 means
-  chapter 1 through 5 should be extracted with the depth
+  The list of colon-separated numbers, chap:level, can
+  change the extraction depth for specific chapters.
+  So 3:2 means that chapter 3 is extracted down to 2 levels
+  (i.e. section level).  You can use a hyphen to specify the
+  range of chapters to set as chapFrom-chapTo:level, so 1-3:5
+  means chapter 1 through 5 should be extracted with the depth
```
Comment (wshito) [inline src/CommandOptions.mjs]: I like to keep the original except the word `collon-separated` in the first line as you proposed.

File: src/CommandOptions.mjs [hunk 3]
```diff
@@ -21,61 +21,61 @@ export const makeConfig = (argv) => {
...
 Example:
-  --depth 2          The default level 2, all the chapters and
+  --depth 2          Set default level 2, all the chapters and
                      sections will be extracted.
-  --depth 3,1:2,8:5  The default level 3, level 2 for Chap 1,
-                     level 5 for Chap 8.
-  --depth 1,3-8:2    The default level 1, level 2 for Chap 3 to 8.
+  --depth 3,1:2,8:5  Set default level 3, level 2 for Ch. 1,
+                     and level 5 for Ch. 8.
+  --depth 1,3-8:2    Set default level 1, and level 2 for Chs. 3 to 8.
```
Comment (wshito) [inline src/CommandOptions.mjs]: Please change 'Set' to 'Sets' here.

**Review by wshito — CHANGES_REQUESTED** [review]
Thank you for your contribution.  I left comments where I prefer the original.

**Pull request merged**